xbmc/filesystem/ZeroconfDirectory.cpp

```diff
@@ -67,8 +65,6 @@ namespace
       fr_protocol = "smb";
     else if(fcr_service_type == "_ftp._tcp.")
       fr_protocol = "ftp";
-    else if(fcr_service_type == "_htsp._tcp.")
-      fr_protocol = "htsp";
```

Comment:
@opdenkamp Just realised that this could have killed your avahi discovery feature for pvr.hts?